augur/__init__.py

```diff
@@ -23,6 +23,7 @@
     "parse",
     "curate",
     "index",
+    "db",
```

Comment:
I placed this here thinking that eventually, it would make sense to load files from previous commands on this list into the database.